src/Relewise.Client.Extensions/Builders.cs

```diff
@@ -149,13 +151,15 @@ private static void MapClientConfig(RelewiseClientOptionsBuilder options, Client
             options.DatasetId = config?.DatasetId;
             options.ApiKey = config?.ApiKey;
             options.Timeout = config?.Timeout;
+            if (config?.ServerUrl != null) options.ServerUrl = config.ServerUrl;
         }
 
         private static void MapClientConfig(RelewiseClientOptionsBuilder options, RelewiseClientOptionsBuilder config)
         {
             options.DatasetId = config.DatasetId;
             options.ApiKey = config.ApiKey;
             options.Timeout = config.Timeout;
+            if (config.ServerUrl != null) options.ServerUrl = config.ServerUrl;
```

Comment:
style: indsæt linje før `if ...`, og skriv dernæst bryd one-liner-if-sætningen op i to linjer:

```

if (config...)
   options.ServerUrl = config.ServerUrl
```